firmware/thruna/BMS/Src/App/App_Eeprom.c

```diff
@@ -129,7 +129,7 @@ EEPROM_StatusTypeDef App_Eeprom_UpdateSavedSocAddress(struct Eeprom *eeprom, uin
     }
     else
     {
-        *address = *address + 1;
+        (*address)++;
```

Comment:
cheeky

Reply:
it wouldn't compile otherwise because 1 is not a uint16_t

Reply:
idk how it got past CI

Reply:
that's weird it might just be a difference in compiler version